recipes/cis/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.3.4" %}
+
+package:
+  name: cis
+  version: {{ version }}
+
+source:
+  fn: cis-{{ version }}.tar.gz
+  url: https://github.com/cedadev/cis/archive/{{ version }}.tar.gz
+  sha256: 553d92ae114da80090125b1b057813e0cd9a4366000d233eb875f43e92fd0ac0
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
@duncanwp I checked and `cis` does use setuptools. See https://github.com/cedadev/cis/blob/master/setup.py#L4

So you can use the `python setup.py install --single-version-externally-managed --record=record.txt`